applications/DEMApplication/python_scripts/DEM_analysis_stage.py

```diff
@@ -228,39 +229,56 @@ def SetRotationalScheme(self):
             sys.exit("\nExecution was aborted.\n")
         return rotational_scheme
 
-    def SetSolverStrategy(self):
-
-        # TODO: Ugly fix. Change it. I don't like this to be in the main...
-        # Strategy object
-        if self.DEM_parameters["ElementType"].GetString() == "SphericPartDEMElement3D" or self.DEM_parameters["ElementType"].GetString() == "CylinderPartDEMElement2D":
-            import sphere_strategy as SolverStrategy
-        elif self.DEM_parameters["ElementType"].GetString() == "SphericContPartDEMElement3D" or self.DEM_parameters["ElementType"].GetString() == "CylinderContPartDEMElement2D":
-            import continuum_sphere_strategy as SolverStrategy
-        elif self.DEM_parameters["ElementType"].GetString() == "ThermalSphericContPartDEMElement3D":
-            import thermal_continuum_sphere_strategy as SolverStrategy
-        elif self.DEM_parameters["ElementType"].GetString() == "ThermalSphericPartDEMElement3D":
-            import thermal_sphere_strategy as SolverStrategy
-        elif self.DEM_parameters["ElementType"].GetString() == "SinteringSphericConPartDEMElement3D":
-            import thermal_continuum_sphere_strategy as SolverStrategy
-        elif self.DEM_parameters["ElementType"].GetString() == "IceContPartDEMElement3D":
-            import ice_continuum_sphere_strategy as SolverStrategy
-        else:
-            self.KRATOSprint('Error: Strategy unavailable. Select a different scheme-element')
-
-        return SolverStrategy
-
-    def SetSolver(self):
+    # def SetSolverStrategy(self):    # TODO should be deleted as soon as posible. check compatibilities
```

Comment:
Just remove things, do not comment them if you do not want to activate them in the future.